src/rpc/server.cpp

```diff
@@ -351,9 +351,7 @@ bool RPCIsInWarmup(std::string *outStatus)
 
 bool IsDeprecatedRPCEnabled(const std::string& method)
```

Comment:
nit: Wouldn't it be better to remove this and replace it with `IsDeprecatedRPCEnabled(gArgs, method)` at the (currently) two call sites?

Reply:
Done